ecommerce/mail_api.py

```diff
@@ -87,7 +87,7 @@ def send_ecommerce_refund_message(order_record):
                 recipient,
                 {
                     "order": order_record,
-                    "readable_id": line.purchased_object.readable_id if line else None,
+                    "readable_id": line.purchased_object.course_number,
```

Comment:
You might want to do the same thing here to check if it's CourseRun before accessing course_number here:
 line.purchased_object.course_number if line and isinstance(line.purchased_object, CourseRun)

Right now it's fine because we don't have the program run as product.